### DISCUSSION TITLE
Why is the cursor so big?

### DISCUSSION THREAD

**Discussion by azizsayfiddin** [keepassxreboot/keepassxc#10627] [Q&A]
OS: arch hyprland
KeePassXC: 2.7.7-2

[Kooha-2024-04-26-14-19-05.webm](https://github.com/keepassxreboot/keepassxc/assets/114720383/b4539aa5-cf66-400d-a980-ae453bd19cd6)

**Comment by azizsayfiddin** [comment]
I ran it using XCURSOR_SIZE=24 and the cursor size is fine there, but when I set it to 27 it gets unevenly large .

**Comment by droidmonkey** [comment]
I'm not sure, but this is a known Linux issue when you mix native apps and flatpak or snap apps.

**Comment by alvilag** [comment]
Same issue here.

OS: debian 12
KeePassXC: 2.7.4 (deb)

**Reply by azizsayfiddin** [reply]
That is the xwayland issue.
